packages/server/graphql/mutations/helpers/bootstrapNewUser.ts

```diff
@@ -18,7 +18,7 @@ import getUsersbyDomain from '../../../postgres/queries/getUsersByDomain'
 import sendPromptToJoinOrg from '../../../utils/sendPromptToJoinOrg'
 import {makeDefaultTeamName} from 'parabol-client/utils/makeDefaultTeamName'
 
-const bootstrapNewUser = async (newUser: User, isOrganic: boolean) => {
+const bootstrapNewUser = async (newUser: User, isOrganic: boolean, searchParams: string) => {
```

Comment:
+1 I'd personally make the param optional so we don't have to change this function invocation where there are no search params/we're not handling it